src/v/kafka/client/client.cc

```diff
@@ -116,6 +116,10 @@ ss::future<> client::do_connect(unresolved_address addr) {
 }
 
 ss::future<> client::connect() {
+    std::random_device rd;
+    std::mt19937 g(rd());
```

Comment:
please use the thread-local random utils.

Reply:
https://github.com/vectorizedio/redpanda/blob/dev/src/v/random/generators.h#L34

we just need to expose the default random engine which i think is mersene twister atm (need to verify) (so same code)

Reply:
topics_frontend just uses
```
        std::shuffle(
          p_as.replicas.begin(),
          p_as.replicas.end(),
          random_generators::internal::gen);
```

Reply:
let's do that